src/magick-image.ts

```diff
@@ -394,6 +397,18 @@ export class MagickImage extends NativeInstance implements IMagickImage {
         });
     }
 
+    brightnessContrast(brightness: number): void;
+    brightnessContrast(brightness: number, contrast: number): void;
```

Comment:
Can you change number into a `Percentage`? And then use `.toDouble()` when calling the native API?